components/form/builder/src/components/button-group.js

```diff
@@ -5,27 +5,32 @@ import MoleculeButtonGroup from '@s-ui/react-molecule-button-group'
 import AtomButtom from '@schibstedspain/sui-atom-button'
 import MoleculeField from '@s-ui/react-molecule-field'
 
-const ButtonGroup = ({errorText, label, id, value, items, onChange}) => {
-  return (
-    <MoleculeField label={label} name={id} errorText={errorText}>
-      <MoleculeButtonGroup type="secondary" fullWidth>
-        {items.map(({name}, i) => {
-          const isSelected = name === value
-          return (
-            <AtomButtom
-              key={i}
-              onClick={() => onChange(name)}
-              focused={isSelected}
-              isButton
-            >
-              {name}
-            </AtomButtom>
-          )
-        })}
-      </MoleculeButtonGroup>
-    </MoleculeField>
-  )
-}
+import WithValidator from '../validatorHoC/WithValidator'
+
+const ButtonGroup = WithValidator(
```

Comment:
Just a suggestion. You can improve the readability separating component from HoC, I mean, for example...
`const ButtonGroup = () => {}`
`export default WithValidator(ButtonGroup)`
What do you think?

Reply:
DONE!